app/src/main/java/com/github/multimatum_team/multimatum/DeadlineDetailsActivity.kt

```diff
@@ -32,40 +34,38 @@ class DeadlineDetailsActivity : AppCompatActivity() {
 
     lateinit var id: DeadlineID
     private var editMode: Boolean = true
+    private val deadlineListViewModel: DeadlineListViewModel by viewModels()
 
     private lateinit var titleView: EditText
     private lateinit var dateView: TextView
     private lateinit var detailView: TextView
     private lateinit var doneButton: CheckBox
 
-    private lateinit var date: LocalDateTime
-    private lateinit var state: DeadlineState
+    // Set them on default value, waiting the fetch of the deadlines
+    private var dateTime: LocalDateTime = LocalDateTime.of(2022, 10, 10, 10, 10)
+    private var state: DeadlineState = DeadlineState.TODO
```

Comment:
Do you really need this? Can't you just leave lateinit?

Reply:
And how about to use a null value to avoid confusion?

Reply:
It is not possible as `LocalDateTime` and `DeadlineState` are non-null type and thus can't have a null value